parcels/particle.py

```diff
@@ -148,6 +176,9 @@ def cptr(i):
             lon = np.linspace(start[0], finish[0], size, dtype=np.float32)
             lat = np.linspace(start[1], finish[1], size, dtype=np.float32)
 
+        if start_field is not None:
+            lon, lat = positions_from_density_field(size, getattr(self.grid, start_field))
```

Comment:
We could... My feeling was that by initialising from a field already in the grid, any type checking or QA that we include in `Grid` object initialisation later on would have already been carried out. Additionally, if we have new particle release events later during the simulation, they can be initialised from a field within the grid that was defined at initialisation, rather than passing around a 'Field' object in the main loop. 
.....But passing the Field object as you suggest doesn't really stop us from doing either of these, so if you think it's more Pythonic to do it this way, then cool! 🐍👍